### PR TITLE
Angular 1.x directive to Angular 1.5 component conversion guide

### PR DESCRIPTION
**Pull request by dgutride**
Describes how to convert an Angular 1.x style directive to a new 1.5  style component

### CODE REVIEW

File: component-conversion.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      scrollSelector: '@',
+      groupHeight: '@',
+      groupClass: '@'
+    },
+    controller: function($window, $timeout) {
+      $scope.toggle = function () {
+  <!------------->
+  <!-- becomes -->
+  <!------------->
+  angular.module('patternfly.notification').directive('pfNotificationDrawer', {
```
Comment (AllenBW) [inline component-conversion.md]: should this be `.component`  rather than `.directive`?

**Comment by dtaylor113** [comment]
Nice @dgutride , LGTM

**Comment by dtaylor113** [comment]
Hi @AllenBW, I think @dgutride is still evaluating 1.6.  I 'believe' when/if we goto 1.6, we will update this guilde.
Thanks!
- Dave

**Comment by dlabrecq** [comment]
LGTM

**Pull request merged**